Deployment/Mirantis-Guidance.md

```diff
@@ -0,0 +1,274 @@
+# Continuation of container support in Azure Service Fabric
+
+## Abstract 
+
+This guidance is for customers using Azure Service Fabric Service with "with Container" VM images restricted to Windows containers on Windows. Service Fabric Product team is following up on the [May 22 targeted deprecation communication](https://portal.microsofticm.com/imp/v3/comms/trackingid/NS72-DZ8) detailing the phase out of “with container” VM image. The May 22 communication was a follow up of the September 2021 [announcement](https://techcommunity.microsoft.com/t5/containers/updates-to-the-windows-container-runtime-support/ba-p/2788799) on the end of support for Mirantis Container Runtime (MCR). 
```

Comment:
This is an internal link, which 3p customers will not have access to.

Reply:
I removed it.